architecture/spmc.rst

```diff
@@ -0,0 +1,267 @@
+====
+SPMC
+====
+
+This document describes the SPMC (S-EL1) implementation for OP-TEE.
+More information on the SPMC can be found in the FF-A specification can be
+found in the
+`FF-A spec <https://developer.arm.com/documentation/den0077/latest>`_.
+
+.. toctree::
+ :numbered:
+
+SPMC Responsibilities
+=====================
+
+The SPMC is a critical component in the FF-A flow. Some of it mayor
+responsibilities are:
+
+- Initialisation and run-time management of the SPs:
+	The SPMC component is responsible for initialisation of the
+	Secure Partitions (Loading the Image, Setting up the stack, HEAP, ...).
+- Routing messages between endpoints:
+	The SPMC is responsible for passing FF-A messages from The Normal World
+	to SPs and back. It also needs to to pass FF-A messages between SPs.
```

Comment:
s/It also needs to to pass/It is also responsible for passing/

Reply:
Done